### ISSUE TITLE
Strange Issue 

### ISSUE THREAD

**Issue by fabltd** [Hieromon/AutoConnect#610]
Hi 

We are having a stange issue. We have devics were the wifi credentials of the AP are not removed. currently to remove the credentials we are calling:

 ```
nvs_flash_erase();
delay(1000);
nvs_flash_init();
```

Is there some other method to reset the list of know SSIDs?

Thanks

**Comment by para360** [comment]
Yes, there is one method that I'm using called esp_wifi_restore(); then reboot the device. It's up to you.

or just add

 config.menuItems = config.menuItems | AC_MENUITEM_DELETESSID to remove a particular SSID from saved credentials.

**Comment by para360** [comment]
If you find any difficulty, then let me know; I'll try to solve it.
